.github/workflows/zaproxy.yml

```
@@ -215,15 +220,134 @@ jobs:
           ZAP_THREAD_PER_HOST: "${{ matrix.thread_per_host }}"
           ZAP_BEFORE_SCRIPT: "${{ matrix.before_script }}"
         run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2193:warning:4:30: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)